packages/replay/test/integration/coreHandlers/handleGlobalEvent.test.ts

```diff
@@ -17,6 +18,10 @@ useFakeTimers();
 let replay: ReplayContainer;
 
 describe('Integration | coreHandlers | handleGlobalEvent', () => {
+  beforeAll(() => {
+    (global as any).TextEncoder = TextEncoder;
```

Comment:
Could we move this into `jest.setup.ts` or similar? Just an idea?

Reply:
+1